packages/react-topology/src/components/defs/__tests__/SVGDefs.spec.tsx

```diff
@@ -16,40 +15,12 @@ describe('SVGDefs', () => {
       id: 'foo',
       children: <span />
     };
-    const wrapper = mount(
+    const { asFragment } = render(
       <SVGDefsContext.Provider value={contextProps}>
         <SVGDefs {...props} />
       </SVGDefsContext.Provider>
     );
-    const innerWrapper = wrapper.find(SVGDefsSetter).first();
-    expect(innerWrapper.props()).toEqual({
-      ...contextProps,
-      ...props
-    });
-  });
-});
-
-describe('SVGDefsSetter', () => {
```

Comment:
Should we open an issue documenting that this test was removed?

Reply:
I'll take another look before doing so, but probably will end up doing that.